Islandora/islandora_videojs
# hls to submodule

**JIRA Ticket**: https://jira.duraspace.org/browse/ISLANDORA-2082

# What does this Pull Request do?

Moves HLS support to a submodule so as to normalize the continued addition of plugins via this method.

# What's new?

* HLS support has been moved to a submodule that needs to be enabled for continued support.
* The admin form and old variable have been removed.
* An update hook has been added to inform of this change.
* Notably, the submodule requires the minified JS instead of the expanded JS.

# How should this be tested?
Should be tested on a site with existing video.js installed, so as to ensure the transition is smooth.

With the submodule set up and installed, HLS playback should function just the same.

# Interested parties
@nhart currently maintains, so other @Islandora/7-x-1-x-committers will have to be roped in on this.
